articles/overview/what-is-qsharp.md

```diff
@@ -0,0 +1,41 @@
+---
+title: What is Q#?
+description: Learn about the Q#, a programming language created by Microsoft to develop applications for quantum computers
+author: natke
+ms.author:  nakersha
+ms.date: 10/16/2019
+ms.topic: article
+uid: microsoft.quantum.overview.qsharp
+---
+
+# What is Q#?
+
+Q# is a programming language created by Microsoft to develop applications for quantum computers.
+
+The Q# programming language takes care of all the quantum computer's internal logic and provides you with an intuitive set of types, operations and logic expressions to develop algorithms without having to worry about the internal logic of the quantum computer.
+
+## We want to code algorithms, not to draw circuits
+
+In [What is quantum computing?](xref:microsoft.quantum.overview.what) we saw that we can write quantum computations as quantum circuits like we do in classical computing. The circuit model has been very useful for many years in quantum computing research.
+
+However, when we write classical programs for normal computers, we don’t think in terms of circuits but in terms of logic.
```

Comment:
```suggestion
However, when writing classical programs for normal computers, we of course no longer think of them in terms of AND/OR gates or assembly language.
```
This (or something similar) would flow nicely into Eduardo/Megan's suggestion below.